### PR TITLE
Enable spotless on *.gradle files

### PR DESCRIPTION
**Pull request by yoomlam**
## What was the problem?

Spotless linter was not being applied to `build.gradle` files.

## How does this fix it?

Add a spotless `groovyGradle` section to all projects

## How to review this PR
Enable "Hide whitespace" when reviewing to see that only formatting has been changed for all files except for file `/build.gradle`:
image

### CODE REVIEW

**Review by wlin2019 — APPROVED** [review]
LGTM

**Comment by yoomlam** [comment]
> I do not think this caught everything. db-init/build.gradle for example does not appear to be linted.

I see -- some subprojects don't apply the `spotless` plugin.
```
❯ g db-init:spotlessApply
...
FAILURE: Build failed with an exception.

* What went wrong:
Task 'spotlessApply' not found in project ':db-init'.
```

Let me see a good way to address this.

File: build.gradle [hunk 1]
```diff
@@ -1,206 +1,207 @@
...
-    def tb = taskBuilder('spotlessCheck')
-    codeModules.each { dependsOn tb(it) }
-}
-
-def spotlessApply = tasks.register('spotlessApply') {
-    def tb = taskBuilder('spotlessApply')
-    codeModules.each { dependsOn tb(it) }
+  // https://axion-release-plugin.readthedocs.io/en/latest/configuration/basic_usage
+  project.version = scmVersion.version
 }
```
Comment (yoomlam) [inline build.gradle:155]: Adding the `spotless` plugin adds these 2 tasks.
(Note: we don't (currently) use pre-commit hooks, so removing these lines has no impact)

File: db-init/build.gradle [hunk 2]
```diff
@@ -3,53 +3,55 @@
...
-            'servicepassword': 'also-not-the-service-password',
-            'adminpassword': 'also-not-the-admin-password'
-    ]
-    configurations = [ 'flywayMigration' ]
+  url = 'jdbc:postgresql://vro:not-the-password@localhost:5432/claims'
+  user = 'vro_user'
+  password = 'not-the-password'
+  locations = [
+    "filesystem:${project.buildDir}/docker/database/migrations"
+  ]
```
Comment (yoomlam) [inline db-init/build.gradle:45]: Fixed by `spotless`.

**Review by au2 — APPROVED** [review]

**Pull request merged**